### PR TITLE
Pin container versions

### LINKED ISSUE #116: Error: Cannot start service wordpress
Creating vip-go-site_wordpress_1 ... error

ERROR: for vip-go-site_wordpress_1  Cannot start service wordpress: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/host_mnt/Users/mike/Sites/test-vip-go/vip-go-site/local/public/wp-cli.yml" to rootfs at "/var/www/html/wp-cli.yml" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

ERROR: for wordpress  Cannot start service wordpress: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/host_mnt/Users/mike/Sites/test-vip-go/vip-go-site/local/public/wp-cli.yml" to rootfs at "/var/www/html/wp-cli.yml" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

ERROR: Encountered errors while bringing up the project.

OSX Catalina 10.15.7 / Docker 3.4.0 / Node 12.22

### PR DESCRIPTION
**Pull request by kasparsd**
Fixes #116.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ services:
 
   # Route HTTP requests to specific containers based on the HTTP_HOST header.
   nginx-proxy:
-    image: jwilder/nginx-proxy
+    image: jwilder/nginx-proxy:0.9.0
```
Comment (kasparsd) [inline docker-compose.yml:7]: Pin to the latest working version of the image.

File: docker-compose.yml [hunk 2]
```diff
@@ -71,6 +71,8 @@ services:
       - .:/var/www/html/wp-content
       - ./local/docker/nginx/config/conf.d:/etc/nginx/conf.d
       - ./local/public/wp:/var/www/html
+      - ./local/public/wp-config.php:/var/www/html/wp-config.php
```
Comment (kasparsd) [inline docker-compose.yml:74]: Sync with the PHP container to ensure it recognizes the mapped files.

File: renovate.json [hunk 3]
```diff
@@ -8,8 +8,7 @@
       "updateTypes": [
         "minor",
         "patch"
-      ],
-      "automerge": true
```
Comment (kasparsd) [inline renovate.json:12]: Don't merge in Docker image updates since we know how that ends :)

**Comment by kasparsd** [comment]
@nandotess Can you please code review this changeset and validate the fix?

**Comment by mehigh** [comment]
This functioned correctly for both me and Dipesh.

**Pull request merged**